src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+
+    public String getDetails() {
+        return ("[" + getStatusIcon() + "] " + description);
+    }
+
+    public void setDone(boolean newStatus) {
+        isDone = newStatus;
```

Comment:
Boolean newStatus could be changed to something that sounds more like a boolean e.g hasDone?

Reply:
Thanks. Will change it in my next commit.